distribution/lib/Standard/Base/0.2.32-SNAPSHOT/src/Data/Vector.enso

```diff
@@ -122,7 +129,7 @@ type Vector
 
              [1, 2, 3, 4].length
     length : Number
-    length = Polyglot.get_array_size this.to_array
+    length = this.store.length
```

Comment:
I'm wondering if there wasn't a specific reason why `Polyglot.get_array_size` was used for arrays here. Like what if the underlying array is not from Java but from other languages, but I'm not sure.
Maybe @kustosz could offer a comment regarding this.

Reply:
I believe it is ok as we should be using the `from_array` method to move polyglot arrays into a Vector (which copies the data across).

That function still does an iterative scan to copy as opposed to Array.Copy so we may want to have a copy based one later.